main.go

```
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+type Headers struct {
+	Accept []string `json:"Accept"`
+	UserAgent []string `json:"User_Agent"`
+}
+
+
+type User struct {
+	Host       string  `json:"host"`
+	UserAgent  string  `json:"user_agent"`
+	RequestUri string  `json:"request_url"`
+	Headers    Headers `json:"headers"`
+}
+var port = ":8081"
+func main() {
+	http.HandleFunc("/", mainPage)
+
+	fmt.Print("Server on port ", port)
+	err := http.ListenAndServe(port, nil)
+	if err != nil {
+		log.Fatal("ListenAndServe", err)
+	}
+}
+
+func mainPage(w http.ResponseWriter, r *http.Request) {
+	headers := Headers{[]string{r.Header.Get("Accept")}, []string{r.Header.Get("User-Agent")}}
+	user := User{r.Host, r.UserAgent(), r.RequestURI, headers}
+	js, _ := json.Marshal(user)
```

Please do not ignore an error message. In case of error you should return it to the client. Do not forget about status code.